### DISCUSSION TITLE
Highlighting flows that are marked

### DISCUSSION THREAD

**Discussion by malwareinfosec** [mitmproxy/mitmproxy#6856] [General]
Hi,

Is it possible to start mitmweb with the highlighting filter turned on for marked flows? Something like: ``mitmweb --view-filter "~marked"``

Optionally, can the same be done via addon scripting?

**Comment by mhils** [comment]
The mitmweb filters are completely client-side at the moment, and we don't have add-on capabilities in JS. But I wouldn't be opposed to a PR that makes the front-end use the view_filter option as the initial value on startup!
